packages/flutter/rust/src/models.rs

```diff
@@ -270,6 +270,7 @@ pub enum _SendPaymentOptions {
     },
     Bolt11Invoice {
         prefer_spark: bool,
+        return_pending_after_secs: Option<u32>,
```

Comment:
Perhaps we can make the naming more intuitive?
wait_for_completion_interval? or anything that focus on the completed payment rather than the pending.

Reply:
👍